packages/node_modules/@webex/plugin-meetings/src/constants.ts

```diff
@@ -1016,16 +1017,23 @@ export const VIDEO_RESOLUTIONS = {
 };
 
 /**
- * Max frame sizes based on h264 configs
+ * Max frame sizes and bandwith constraints based on h264 configs
  * https://en.wikipedia.org/wiki/Advanced_Video_Coding
  */
-export const MAX_FRAMESIZES = {
-  [QUALITY_LEVELS.LOW]: 1620,
-  [QUALITY_LEVELS.MEDIUM]: 3600,
-  [QUALITY_LEVELS.HIGH]: 8192
+export const REMOTE_VIDEO_CONSTRAINTS = {
+  MAX_FS: {
+    [QUALITY_LEVELS.LOW]: 1620,
+    [QUALITY_LEVELS.MEDIUM]: 3600,
+    [QUALITY_LEVELS.HIGH]: 8192
+  },
+  // Decoder's maximum processing speed in macroblocks per second
+  MAX_MBPS: {   
+    [QUALITY_LEVELS.LOW]: 27600,
```

Comment:
@arun3528 could you please provide your input here? Also it would be helpful if you could provide some info on how to calculate these values as I couldn't find that on https://en.wikipedia.org/wiki/Advanced_Video_Coding

@marcin-bazyl What value do you suggest?